### PR TITLE
`<LinearProgressBar/>`- adding success skin and update existing colors

### PR DESCRIPTION
**Pull request by TalZa1810**
### 🔦 Summary

Resolves https://github.com/wix/wix-style-react/issues/4181 

- Adding prop `skin` to the component which can be one of the following: `standard` / `success`.
- Adding a function to both driver & uni-driver `getSkin( )` and updating unit tests accordingly.
- Updating visual tests with the new skin. 

### ✅ Checklist

- [x] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [x] Story
  - [x] API description
  - [x] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [x] Component
  - [x] Visual test

### CODE REVIEW

**Comment by TalZa1810** [comment]
Hi @LittleTal ,
Can you please review this?

Thanks :)

**Comment by TalZa1810** [comment]
Changing the text (progress indication) will be in a different PR (`wix-ui-core`).

**Comment by TalZa1810** [comment]
Hi @nitayneeman ,
Can you please review this PR?
The only thing I wasn't sure is about the names of the skins- `success` & `completed` since we have a `successIcon` prop which may be confusing since we use the icon for both `success` & `completed` skins. Wdyt? 

The visual tests are failing since it's a breaking change. After the review, I will approve those change in applitools.

**Comment by TalZa1810** [comment]
Related PR for changing the progress indicator to be white when `light` theme is chosen:
https://github.com/wix/wix-ui/pull/1651

This PR should be merged after the `core` PR is merged.

File: src/LinearProgressBar/LinearProgressBar.st.css [hunk 1]
```diff
@@ -72,12 +75,16 @@
   font-family: value(fontMedium);
 }
 
-.root:success::indicationContainer {
-  background: value(B10);
+.root:skin(completed):success::indicationContainer {
```
[the remark this answers is not in the document]
Comment (TalZa1810) [reply src/LinearProgressBar/LinearProgressBar.st.css]: It's a bit confusing. The `success` here is not the `success` skin but the the `success` state, meaning the linear progress bar value is 100% without any errors (of both `success` & `standard` skins.  The `indicationContainer` div is the div of the icon / progress indication.

File: src/LinearProgressBar/LinearProgressBar.js [hunk 2]
```diff
@@ -44,8 +48,10 @@ class LinearProgressBar extends React.PureComponent {
     return (
       <CoreLinearProgressBar
         data-hook={dataHook}
-        {...styles('root', { light }, this.props)}
+        {...styles('root', { light, skin }, this.props)}
+        data-skin={skin}
```
Comment (nitayneeman) [inline src/LinearProgressBar/LinearProgressBar.js]: Maybe I'm missing something, but why do we need this data-attribute (Stylable adds its own attributes either way, so `getStyleState` is supposed to be OK without that)?

**Comment by TalZa1810** [comment]
@nitayneeman  Fixed your comments.
@LittleTal I've changed the progress indication color in this PR (both when it is light and regular). 

Let me know if there are more comments or I can merge this :)

**Comment by nitayneeman** [comment]
> @nitayneeman Fixed your comments.
> @LittleTal I've changed the progress indication color in this PR (both when it is light and regular).
> 
> Let me know if there are more comments or I can merge this :)

I just added a small question above :)

File: src/LinearProgressBar/LinearProgressBar.js (hunk 2 above)
Comment (TalZa1810) [reply src/LinearProgressBar/LinearProgressBar.js]: The other option is using `getAttribute` for the wrapper element which seems a bit "hacky". 
I've pushed it to this PR.

**Comment by TalZa1810** [comment]
@nitayneeman 
It fails on unrelated applitools test. The only test which is kinda related is `SidebarHeader` visual test which uses `LinearProgressBar` component. 
Should I refactor the visual test in this PR? 

https://github.com/wix/wix-style-react/blob/master/src/SidebarHeader/test/SidebarHeader.visual.js#L48

Refactor the test so that it uses `<LinearProgressBar light/>` for the dark skin and the other one for the white skin?

**Comment by TalZa1810** [comment]
Fails on visual tests which are not related to the change.

**Pull request merged**